openmmtools/states.py

```diff
@@ -2066,25 +2066,26 @@ def __getstate__(self):
         """Return a dictionary representation of the state."""
         # Create original ThermodynamicState to serialize.
         thermodynamic_state = object.__new__(self.__class__.__bases__[1])
-        thermodynamic_state.__dict__ = copy.deepcopy(self.__dict__)
+        thermodynamic_state.__dict__ = self.__dict__
```

Comment:
I removed it because `utils.serialize` doesn't modify the passed arguments, and doing a `copy.deepcopy` involves copying a whole system, which may take time. It's just a small optimization.